--- perf/test.pl.orig	2014-05-05 09:32:39.171338467 +0100
+++ perf/test.pl	2014-05-05 11:20:43.471589649 +0100
@@ -61,7 +61,7 @@
 for ( my $tid = 1; $tid <= 45; $tid++ ) {
     last;
     $ENV{SGE_TASK_ID} = $tid;
-    system( "perl t/Bio-Grid-Run-SGE-Master.script.pl -w $td/master_config$job_id 2>/dev/null"  );
+    system( "$^X t/Bio-Grid-Run-SGE-Master.script.pl -w $td/master_config$job_id 2>/dev/null"  );
 
     test_atomic($job_id, $tid-1, $tid-1, $seqs);
 
@@ -86,7 +86,7 @@
 for ( my $tid = 1; $tid <= 23; $tid++ ) {
     last;
     $ENV{SGE_TASK_ID} = $tid;
-    system( "perl t/Bio-Grid-Run-SGE-Master.script.pl -w $td/master_config$job_id 2>/dev/null"  );
+    system( "$^X t/Bio-Grid-Run-SGE-Master.script.pl -w $td/master_config$job_id 2>/dev/null"  );
     test_atomic($job_id, $tid-1, $tid - 1, $seqs);
     test_atomic($job_id, 22+$tid, 22+$tid , $seqs) if(22+$tid < 45);
 }
@@ -112,7 +112,7 @@
 
 for ( my $tid = 1; $tid <= 15; $tid++ ) {
     $ENV{SGE_TASK_ID} = $tid;
-    say( "perl t/Bio-Grid-Run-SGE-Master.script.pl -w $td/master_config$job_id 2>/dev/null"  );
+    say( "$^X t/Bio-Grid-Run-SGE-Master.script.pl -w $td/master_config$job_id 2>/dev/null"  );
     last;
 
     test_atomic($job_id, ($tid*3)-3, ($tid*3)-3, $seqs);
